pavelib/prereqs.py

```diff
@@ -31,6 +31,11 @@
     'requirements/edx/post.txt',
 ]
 
+# PhilU requirements
+PHILU_REQ_FILES = 'requirements/philu/base.txt'
```

Comment:
what about adding a line(`'requirements/philu/base.txt'`)  to `PYTHON_REQ_FILES` list above.